packages/admin-stories/src/admin/table/TableWithFilterBar.tsx

```diff
@@ -0,0 +1,160 @@
+        email: "email",
+        username: "username",
+        name: "name",
+        website: "website",
+    };
+
+    const q = Object.keys(args).reduce((acc: { [key: string]: any }, key: string): { [key: string]: any } => {
+        if (paramMapping[key] && args[key]) {
+            acc[paramMapping[key]] = args[key];
+        }
+        return acc;
+    }, {});
+
+    return `users?${qs.stringify(q, { arrayFormat: "brackets" })}`;
+}
+
+interface IQueryData {
+    users: Array<{
+        id: number;
+        name: string;
+        username: string;
+        email: string;
+        phone: string;
+        website: string;
+    }>;
+}
+
+interface IFilterValues {
+    username: any;
+    name: any;
+    email: any;
+    website: any;
+}
+
+interface IVariables extends IFilterValues {
+    pathFunction: any;
+}
+
+const ExampleWithSelect: React.FC = () => {
+    const options = [
+        { value: "Sincere@april.biz", label: "Sincere@april.biz" },
+        { value: "Shanna@melissa.tv", label: "Shanna@melissa.tv" },
+        { value: "Nathan@yesenia.net", label: "Nathan@yesenia.net" },
+    ];
+    return <Field name="email" type="text" component={FinalFormReactSelectStaticOptions} fullWidth options={options} />;
+};
+
+function Story() {
+    const filterApi = useTableQueryFilter<Partial<IFilterValues>>({});
+
+    const { tableData, api, loading, error } = useTableQuery<IQueryData, Partial<IVariables>>()(query, {
+        variables: {
+            email: filterApi.current.email?.email,
+            name: filterApi.current.name?.name,
+            username: filterApi.current.username?.username,
+            website: filterApi.current.website?.url,
+            pathFunction,
+        },
+        resolveTableData: (data) => ({
+            data: data.users,
+            totalCount: data.users.length,
+        }),
+    });
+
+    return (
+        <TableQuery api={api} loading={loading} error={error}>
+            <TableFilterFinalForm filterApi={filterApi}>
+                <Typography variant="h5">FilterBar</Typography>
+                <FilterBar>
+                    <FilterBar.PopOverFormField label={"Username"} name="username" fieldBarWidth={150}>
+                        <Field name="username" type="text" component={FinalFormInput} fullWidth />
+                    </FilterBar.PopOverFormField>
+                    <FilterBar.MoreFilterButton fieldBarWidth={150}>
+                        <FilterBar.PopOverFormField label={"Email"} name="email" fieldBarWidth={150}>
+                            <ExampleWithSelect />
+                        </FilterBar.PopOverFormField>
+                        <FilterBar.PopOverFormField label={"Name"} name="name" fieldBarWidth={150}>
+                            <Field name="name" type="text" component={FinalFormInput} fullWidth />
+                        </FilterBar.PopOverFormField>
+                        <FilterBar.PopOverFormField label={"Website"} name="website" fieldBarWidth={150}>
```

Comment:
at the beginning it was used to calculate the available space on the screen (or move to sidebar), but now it is only used so all fields have the same width :) i will remove it because i also think its probably not needed anymore and should be custom styleable :)